### PR TITLE
Peacock tails!

### PR DESCRIPTION
**Pull request by rboys2**
Prettiebyrd fluff tails, 
https://forum.vore-station.net/viewtopic.php?f=26&t=1914

### CODE REVIEW

File: code/modules/vore/appearance/sprite_accessories_vr.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+ing /*
```
Comment (Novacat) [inline code/modules/vore/appearance/sprite_accessories_vr.dm:1]: Oopsie!

**Review by Novacat — CHANGES_REQUESTED** [review]

File: code/modules/vore/appearance/sprite_accessories_vr.dm (hunk 1 above)
Comment (TheFurryFeline) [reply code/modules/vore/appearance/sprite_accessories_vr.dm:1]: And yet it compiles, apparently. 🤔

Comment (Novacat) [reply code/modules/vore/appearance/sprite_accessories_vr.dm:1]: It should still be fixed.

Comment (TheFurryFeline) [reply code/modules/vore/appearance/sprite_accessories_vr.dm:1]: I know, just amusing that it doesn't trigger the bot.

Comment (TheFurryFeline) [reply code/modules/vore/appearance/sprite_accessories_vr.dm:1]:
```suggestion
/*
```

**Review by Novacat — APPROVED** [review]

**Pull request merged**